pkg/model/common.proto

```diff
@@ -22,7 +22,8 @@ import "validate/validate.proto";
 enum ApplicationKind {
     KUBERNETES = 0;
     TERRAFORM = 1;
-    CROSSPLANE = 2;
```

Comment:
Should we remove it or keep this with a comment?

Reply:
yeh, I commented it with a note (TODO). Is that good enough? We have to comment out this value or it would be reflected web client application kind components and we have to add conditions to filter out the value.

Reply:
I got it. 👍